python/default_crab_config.py

```diff
@@ -14,13 +14,9 @@ def create_config(is_mc):
     config.General.transferLogs = True
 
     config.JobType.pluginName = 'Analysis'
-    config.JobType.psetName = '../python/dummy_pset.py'
-    config.JobType.scriptExe = '../bin/runFrameworkOnGrid.sh'
+    config.JobType.psetName = '../../Framework/test/TestConfiguration.py'
     config.JobType.sendPythonFolder = True
-    config.JobType.disableAutomaticOutputCollection = True
     config.JobType.allowUndistributedCMSSW = True
-    config.JobType.inputFiles = ['../python/runFrameworkOnGrid.py']
```

Comment:
Really? AFAIK there's nothing we need to upload other than the pset, and it's done automatically. Do you see something else or are you talking about crab forcing an empty inputFiles array in the config ?